lychee-bin/src/formatters/stats/markdown.rs

```diff
@@ -67,10 +67,9 @@ fn stats_table(stats: &ResponseStats) -> String {
 /// Optional details get added if available.
 fn markdown_response(response: &ResponseBody) -> Result<String> {
     let mut formatted = format!(
-        "* [{}] [{}]({})",
+        "* [{}] <{}>",
```

Comment:
Why the angular brackets? I thought we wanted to go for the "plain" format? E.g. `"{}"` instead of `<{}>`?

Reply:
angle brackets create [clickable links in markdown](https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses) and they also still work in GitHub actions (tested with a [simple action](https://github.com/Aariq/test-renv-cache/actions/runs/13377077841/job/37358471928))

Reply:
Github may support automatically detecting links and making them clickable, but I think it's still better to explicitly state, that it's a link.

And I'm not sure, if stuff like https://example.com/myendingpunctuation. Would be correctly displayed

Reply:
Sounds convincing. 👍 Thanks for the explanation.